### PR TITLE
Adding installation doc for ubuntu16

### PR DESCRIPTION
**Pull request by quqinghai**
Adding a simple installation doc for ubuntu16/debian platform.

### CODE REVIEW

File: Installation_ubuntu16.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+# 1. install nvm/node
+```
+$ cd ~/
+$ git clone https://github.com/creationix/nvm.git .nvm
+$ cd ~/.nvm 
+$ git checkout v0.33.8
```
Comment (twd2) [inline Installation_ubuntu16.md]: Magic number

File: Installation_ubuntu16.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh" 
+[ -s "$NVM_DIR/bash_completion" ] && \. "$NVM_DIR/bash_completion"
+EOF
+$ cd ~/
+$ nvm ls-remote
+$ nvm install --lts=carbon 
+$ node -v 
+```
+
+# 2. install mongodb/rabbitmq
```
Comment (twd2) [inline Installation_ubuntu16.md]: "/" means *and*?

File: Installation_ubuntu16.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+$ curl "http://geolite.maxmind.com/download/geoip/database/GeoLite2-City.mmdb.gz" | gunzip -c > GeoLite2-City.mmdb 
+$ npm run build:production
+$ python -m vj4.server --listen=http://0.0.0.0:8888/
+$ python -m vj4.model.user add -1 your_username your_password user@domain.com 
+$ python -m vj4.model.user set_superadmin -1 
+$ python -m vj4.model.user add -2 judge 123456 user@domain.com
+$ python -m vj4.model.user set_judge -2
+$ python -m vj4.job.rp recalc_all
+$ python -m vj4.job.rank run_all
+$ python -m vj4.server --listen=http://0.0.0.0:8888/ --smtp-host=your_smtp_server --smtp-user=your_smtp_user --smtp-password=your_smtp_password
```
Comment (twd2) [inline Installation_ubuntu16.md]: mail-from=xxx

File: Installation_ubuntu16.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+
+# 4. install jd4
+```
+$ git clone https://github.com/vijos/jd4.git
+$ sudo python -m pip install -r requirements.txt
+$ mkdir -p ~/.config/jd4
+$ cp examples/config.yaml ~/.config/jd4/
+$ cp examples/langs.yaml ~/.config/jd4/
+$ sudo python setup.py build_ext --inplace
```
Comment (twd2) [inline Installation_ubuntu16.md]: `sudo` is not needed?

File: Installation_ubuntu16.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+$ sudo python -m jd4.daemon
```
Comment (twd2) [inline Installation_ubuntu16.md]: `sudo` is only needed on first execution?

File: Installation_ubuntu16.md (hunk 2 above)
Comment (quqinghai) [reply Installation_ubuntu16.md]: yes.

File: Installation_ubuntu16.md (hunk 1 above)
Comment (quqinghai) [reply Installation_ubuntu16.md]: v0.33.8？

**Comment by breeswish** [comment]
I suggest to add them to our doc repo: https://github.com/vijos/about-vijos

File: Installation_ubuntu16.md (hunk 1 above)
Comment (breeswish) [reply Installation_ubuntu16.md]: nvm is not a required component. Following the official Nodejs installation links README given is enough.

File: Installation_ubuntu16.md (hunk 2 above)
Comment (breeswish) [reply Installation_ubuntu16.md]: Normally `/` means "or". You should use the word "and" here.

**Comment by twd2** [comment]
Not active for one year, closing. Feel free to reopen if you have any update.

**Pull request closed without merging**